packages/dashboard/e2e/tests/widgets/line.spec.ts

```diff
@@ -0,0 +1,546 @@
+import { expect, test } from '../test';
+import { getTrendCursorText } from './LineWidget';
+import { Locator } from '@playwright/test';
+
+const NEW_Y_LABEL = 'This is new Y-axis Label';
+const Y_MIN = '1';
+const Y_MAX = '99';
+
+const getComputedStyles = async (locator: Locator) => {
+  return await locator.evaluate((el) => {
+    const computedStyle = window.getComputedStyle(el);
+    return {
+      strokeWidth: computedStyle.getPropertyValue('stroke-width'),
+      strokeDashArray: computedStyle.getPropertyValue('stroke-dasharray'),
+    };
+  });
+};
+
+//  -----------------------------FOR TEST FAILURES --------------------------------------
+// 1. check if the ECHARTS is rendering the svg in a different fashion, check its version number and the rendered svg
+// 2. if it's in the following fashion, check further in the test case, else you may have to update wherever we are
+//    testing the svg of the rendered chart
+//  ---------------------------ECHARTS SVG STRUCTURE-------------------------------------
```

Comment:
love this! really good guide to have in case echarts changes their rendering pattern and we need to update the tests 🙏 tytyty